### PR TITLE
Add missing HelixEmote fields

### PR DESCRIPTION
**Pull request by zunderscore**
Type: Improvement

Fixes: #426

Adds missing fields to HelixEmote base object, supported by all 3 emote calls: global, channel, and set emotes.

### CODE REVIEW

File: packages/api/src/api/helix/chat/HelixEmote.ts [hunk 1]
```diff
@@ -28,4 +55,45 @@ export class HelixEmote extends DataObject<HelixEmoteData> {
...
+	 *
+	 * @param scale The scale of the image.
+	 */
+	getAnimatedImageUrl(scale: HelixEmoteScale = '1.0'): string | null {
+		if (this[rawDataSymbol].format.includes('animated') && this[rawDataSymbol].scale.includes(scale)) {
+			this.getFormattedImageUrl(scale, 'animated');
+		}
+
+		return null;
+	}
```
Comment (d-fischer) [inline packages/api/src/api/helix/chat/HelixEmote.ts:83]: Couldn't these methods also have an optional `themeMode` parameter?

Comment (zunderscore) [reply packages/api/src/api/helix/chat/HelixEmote.ts]: Added

File: packages/api/src/api/helix/chat/HelixEmote.ts [hunk 2]
```diff
@@ -28,4 +55,47 @@ export class HelixEmote extends DataObject<HelixEmoteData> {
...
+
+	/**
+	 * Gets the URL of the emote image in static format at the given scale and theme mode, or null if a static emote image at that scale/theme mode doesn't exist.
+	 *
+	 * @param scale The scale of the image.
+	 * @param themeMode The theme mode of the image, either `light` or `dark`.
+	 */
+	getStaticImageUrl(scale: HelixEmoteScale = '1.0', themeMode: HelixEmoteThemeMode = 'light'): string | null {
+		if (this[rawDataSymbol].format.includes('static') && this[rawDataSymbol].scale.includes(scale)) {
+			this.getFormattedImageUrl(scale, 'static', themeMode);
```
Comment (d-fischer) [inline packages/api/src/api/helix/chat/HelixEmote.ts:67]: This doesn't return anything, and resulting from that, the method always returns `null`. (same for the other one)

**Review by d-fischer — CHANGES_REQUESTED** [review]

File: packages/api/src/api/helix/chat/HelixEmote.ts (hunk 2 above)
Comment (zunderscore) [reply packages/api/src/api/helix/chat/HelixEmote.ts:67]: Fixed.

**Review by d-fischer — APPROVED** [review]

**Pull request merged**